src/Sylius/Bundle/CoreBundle/Doctrine/Migrations/AbstractMySqlMigration.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+abstract class AbstractMySqlMigration extends AbstractMigration
```

Comment:
Be consistent with Doctrine:
```suggestion
abstract class AbstractMySQLMigration extends AbstractMigration
```